workshops/drawing_shapes_workshop/README.md

```diff
@@ -0,0 +1,309 @@
+---
+name: 'Drawing Shapes Program'
+description: 'Make a drawing program with shapes!'
+author: '@JakeGerber'
+---
+
+# Create a Drawing Program with Shapes!
+
+<img src="https://cloud-r8adu66vb.vercel.app/0screenshot__1391_.png" width="580" alt="Drawing Example">
+
+<img src="https://thumbs.gfycat.com/FantasticFaintAmurminnow-small.gif" width="380" alt="Nice Gif">
+
+# Repl.it Setup
+
+We're going to use [Repl.it](https://repl.it/~) to create the project. It is an online IDE!
+
+Create a new repl and use Python as the language.
```

Comment:
Many of my club members don't know how to do this, could you please give more context for people like them?